src/open_inwoner/pdc/models/category.py

```diff
@@ -72,14 +72,6 @@ class Category(MP_Node):
             "Zaaktypen waarvoor bij aanwezigheid dit onderwerp getoond moet worden."
```

Comment:
Graag valideren of de featureflag aan staat, alleen het vullen van zaaktypen toestaan indien deze is geactiveerd.